cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -1895,25 +1904,110 @@ def test_export_can_reuse_older_file_if_still_relevant(self):
         mock_osp_exists.assert_called_with(first_export_path)
         mock_os_replace.assert_not_called()
 
+    def test_initiate_concurrent_export_by_different_users(self):
+        side_effect = self.side_effect
+        chain_side_effects = self.chain_side_effects
+        process_closing = self.process_closing
+
+        format_name = "CVAT for images 1.1"
+
+        def _export(
+            *_, task_id: int, result_queue: multiprocessing.Queue, sleep_on_export: bool = False
+        ):
+            from os import replace as original_replace
+
+            from cvat.apps.dataset_manager.task import export_task as original_export_task
+
+            lock_ttl = 2
+            lock_acquire_timeout = lock_ttl * 2
+
+            with (
+                patch("cvat.apps.dataset_manager.views.EXPORT_LOCK_TTL", new=lock_ttl),
+                patch(
+                    "cvat.apps.dataset_manager.views.EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT",
+                    new=lock_acquire_timeout,
+                ),
+                patch(
+                    "cvat.apps.dataset_manager.views.get_export_cache_lock",
+                    new=self.patched_get_export_cache_lock,
+                ),
+                patch("cvat.apps.dataset_manager.views.os.replace") as mock_os_replace,
+                patch(
+                    "cvat.apps.dataset_manager.views.rq.get_current_job"
+                ) as mock_rq_get_current_job,
+                patch("cvat.apps.dataset_manager.views.django_rq.get_scheduler"),
+                patch("cvat.apps.dataset_manager.views.task.export_task") as mock_export_task_func,
+            ):
+                mock_export_task_func.side_effect = chain_side_effects(
+                    original_export_task,
+                    side_effect(sleep, lock_acquire_timeout + 1 if sleep_on_export else 0),
```

Comment:
What is the point of this `sleep`? Consider using conditions instead. Using simple sleeps implies unnecessary performance requirements on the executing system, it leads to unstable tests.